src/internet_identity/internet_identity.did

```diff
@@ -775,6 +775,7 @@ type AccountInfo = record {
     last_used : opt Timestamp;
     // Configurable properties
     name : opt text;
+    is_default : opt bool;
```

Comment:
@danblackadder, this is how the FE can tell which account is the default. 

In case this field is `null`, I suggest assuming that the default account = the primary account.